homeassistant/components/binary_sensor/wink.py

```diff
@@ -61,7 +61,7 @@ def _pubnub_update(self, message, channel):
                 json_data = message
             self.wink.pubnub_update(json.loads(json_data))
             self.update_ha_state()
-        except (AttributeError, KeyError):
+        except (AttributeError, KeyError, AttributeError):
```

Comment:
@MartinHjelmare You're right thanks for the help. I was running a different branch without AttributeError being caught and didn't realize it, of course when I made this PR I started the branch from dev which already had the fix. I guess that is what I get for rushing to fix it. :disappointed: